client/components/recycleAndReuseComponents/MultiSelect.js

```diff
@@ -1,28 +1,58 @@
-import { Button } from "@chakra-ui/react";
-import React from "react";
-import Select from "react-select";
-import { options }  from '../../../mockData/data';
+import { Button } from '@chakra-ui/react'
+import React from 'react'
+import Select from 'react-select'
+import { options }  from '../../../mockData/data'
 
-const MultiSelect = ({ optionss = options, setInput }) => {
-  const handleChange = (event) => {
-    setInput({ input: event });
-  };
+const MultiSelect = ({ data, setItems, navigateToVerifyItem }) => {
+  const parseItemsIntoOptions = (categories, items) => {
+    const allOptions = []
+    categories.forEach(category => {
+      const itemsInCategory = items.filter(item => item.category === category)
+      const itemOptions = []
+      itemsInCategory.forEach(itemInCategory => {
+        const option = {
+          "label": itemInCategory.description,
+          "value": itemInCategory.id
+        }
+        itemOptions.push(option)
+      })
+
+      const categoryOptions = {
+        "label": category,
+        "options": itemOptions
+      }
+      allOptions.push(categoryOptions)
+    })
+    
+    return allOptions
+  }
```

Comment:
Currently, the [mock data](https://api.npoint.io/a8416aa207861acb363d) is directly formatted for display in the `MultiSelect` component. However, this mock data does not allow for other data like whether an item is blue-bin recyclable etc. 

I've created [another dummy dataset](https://www.npoint.io/docs/324727202a8ce24af12f). This allows for the same dataset to be reused across all components, although it also means that the data has to be parsed before it can be displayed.